src/pages/admin/training/list/components/spec/trainigTable.spec.tsx

```diff
@@ -24,11 +24,15 @@ describe('TrainingTabletComponent', () => {
         id: 2,
```

Comment:
Typo in the file it should be _trainingTable.tsx_  (missing "n" on the file name)